recipe/meta.yaml

```
@@ -33,9 +32,11 @@ build:
 requirements:
   build:
     - python
+    # run-dep as a workaround for now; upstream package still imports from distutils
+    - setuptools  # [py>311]
     - {{ posix }}bison
     - {{ posix }}flex
-    - meson >=0.58.2
+    - meson >=1.2.0
```

```suggestion
    - meson >=0.6.0
```
I see this [here](https://gitlab.gnome.org/GNOME/gobject-introspection/-/blob/1.78.1/meson.build?ref_type=tags#L3)

---

Thank you